e2e/components/Slider/Slider-test.avt.e2e.js

```diff
@@ -92,6 +92,20 @@ test.describe('@avt Slider', () => {
     await expect(page).toHaveNoACViolations('Slider-with-layer');
   });
 
+  test('@avt-advanced-states slider with custom format', async ({ page }) => {
+    await visitStory(page, {
+      component: 'Slider',
+      id: 'components-slider--slider-with-custom-value-label',
+      globals: {
+        theme: 'white',
+      },
+    });
+
+    await page.keyboard.press('Tab');
+    await expect(page.getByRole('slider')).toHaveValue('Medium');
+    await expect(page).toHaveNoACViolations('Slider-with-custom-value-label');
```

Comment:
For @avt-advanced-states we just have to keep the line below.
```suggestion
    await expect(page).toHaveNoACViolations('slider-with-custom-value-label');
```

To add keyboard navigation you can use the `@avt-keyboard-nav` and add the tests.

I would suggest the following tests for keyboard navigation:
- Tab to the Slider to verify focused element and it's value "Medium".
- After pressing Shift+ArrowRight the value should be "High"
- After pressing two times Shift+ArrowLeft the value should be "Low"

 `await page.keyboard.press('Shift+ArrowRight');`

Reply:
> Add that story to AVT and VRT - each of these should first move focus to one of the slider handles (so a tooltip is visible) before running toBeAccessible or snapshotStory

I assumed we have to move focus before we run tests. I'll change that. Thank you :)